src/20231207_AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership/AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207.t.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {ProtocolV3TestBase} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+
+import {AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207} from './AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207.sol';
+
+/**
+ * @dev Test for AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207
+ * command: make test-contract filter=AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207
+ */
+contract AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207_Test is
+  ProtocolV3TestBase
+{
+  AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207 internal proposal;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 18736059);
+    proposal = new AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnership_20231207();
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_execute() public {
+    (, , , , , , uint256 streamToCancelRemaining, ) = AaveV3Ethereum.COLLECTOR.getStream(
+      proposal.STREAM_TO_CANCEL()
+    );
+
+    assertGt(streamToCancelRemaining, 0);
+
+    vm.startPrank(proposal.STREAM_TWO_RECEIVER());
+    AaveV3Ethereum.COLLECTOR.withdrawFromStream(
```

Comment:
For what is this?

Reply:
Cancel existing stream, to test that the available funds are moved.

Reply:
Reference: https://governance.aave.com/t/arfc-tokenlogic-hohmann-transfer/15051

Reply:
Was meaning about the need to test this, but it is fine. Resolved